docs/topics/proc_deploying-an-application-to-standalone-rhel.adoc

```diff
@@ -0,0 +1,121 @@
+// This is a parameterized module. Parameters used:
+//
+// :parameter-jar-file-name: runtime-specific default target Maven build artifact name.
+// :parameter-version-runtime-plugin: runtime-specific name for the runtime version property variable in the POM file.
+// :parameter-artifact-name-runtime-plugin: runtime plugin name in the POM file.
+// :parameter-group-name-runtime-plugin: runtime plugin Maven artifact group name.
+// :parameter-runtime-name: Runtime specific atribute for substituting the approrpiate expanded rumtime name.
+//  context: used in anchor IDs to conflicts due to duplicate IDs.
+//
+// Rationale: This procedure is the same for 2 or more runtimes.
+[id='deploying-an-application-to-standalone-rhel_{context}']
+= Deploying an application to standalone Red Hat Enterprise Linux
+
+You can deploy a {parameter-runtime-name} application to Red Hat Enterprise Linux.
+
+.Prerequisites
+
+* A Maven-based application.
```

Comment:
I would add a mention of the runtime here to imply we expect they're already using it.

Reply:
Ok, let me add that, too